packages/react-theming/README.md

```diff
@@ -365,11 +376,33 @@ downlevel slots.
 
 ### Slots
 
-TODO: Describe how to interact with slots
+`props.slots` will be a dictionary mapping individual slots to
+`ReactType` values which can then be used in JSX.
+
+`slots` are defined in createComponent by specifying a slots argument:
+
+```jsx
+const CreatedComponent = createComponent(BaseComponent, {
+  slots: {
+    root: 'div',
+    button: MyLibrary.FancyButton
+  }
+});
+
+```
 
 ### Slot Props
 
-TODO: Describe how to interact with slotProps
+`props.slotProps` defines the props that are distributed to each slot.
```

Comment:
We discussed multiple times about replacing `slotProps` with direct props on the component. The slots prop is a great idea, but for the slotProps I still don't get the benefits we would get from it.

Reply:
The main advantages are to easily replace the implementation without having to re-handle all of the props. I think we agree of having that interface at the top level, but for a base component I'm not sure it makes sense.